cts/environment.py

```diff
@@ -88,11 +87,6 @@ def SeedRandom(self, seed=None):
         if not seed:
             seed = int(time.time())
 
-        if self.has_key("RandSeed"):
-            self.logger.log("New random seed is: " + str(seed))
-        else:
-            self.logger.log("Random seed is: " + str(seed))
-
```

Comment:
This we want. In theory we could want to play back the same CTS run - which we could do if we have the same seed.